.gitignore

```diff
@@ -7,7 +7,6 @@
 # build artifacts
 coverage
 opa_*
-.Dockerfile_*
```

Comment:
Can we keep this but include a comment indicating these files are deprecated/unused? I foresee a bunch of people submitting PRs that accidentally include these files.